src/main/java/uk/gov/companieshouse/acsp/service/AcspService.java

```diff
@@ -98,6 +98,14 @@ public ResponseEntity<Object> updateACSPDetails(Transaction transaction,
             throw new InvalidTransactionStatusException(String.format(
                     "Can't update transaction with stastus: %s ", transaction.getStatus().toString()));
         }
+
+        if (acspDataDto.getCompanyDetails() != null) {
```

Comment:
What about unincorporated and Sole trader scenario, the company name is not saved in companyDetails for this